.devcontainer/development/Dockerfile

```
@@ -84,11 +84,11 @@ RUN npm install -g playwright@1.56.0 && \
 # Install Playwright browsers (Chromium, Firefox) with system dependencies
 # Note: Chrome excluded - no ARM64 Linux builds (breaks Apple Silicon). Chromium is equivalent.
 # Version 1.56.0+ correctly uses t64 package names (e.g., libasound2t64) for Ubuntu 24.04
-RUN npx playwright install --with-deps chromium firefox && \
+RUN apt-get update || true && npx playwright install --with-deps chromium firefox && \
```

Hadolint (2.14.0)

[info] Delete the apt lists (/var/lib/apt/lists) after installing something

(DL3009)